lightning/src/ln/channelmanager.rs

```diff
@@ -913,6 +922,7 @@ impl<ChanSigner: ChannelKeys, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 			// the latest local state, which is the best we can do anyway. Thus, it is safe to
 			// ignore the result here.
 			let _ = self.chain_monitor.update_channel(funding_txo, monitor_update);
+			self.persistence_notifier.notify();
```

Comment:
This is an internal function, I'd think the calls here need to happen at the callsite.